components/actionbutton/stories/actionbutton.stories.js

```diff
@@ -1,8 +1,10 @@
 import { default as IconStories } from "@spectrum-css/icon/stories/icon.stories.js";
+import { Sizes } from "@spectrum-css/preview/decorators";
 import { disableDefaultModes } from "@spectrum-css/preview/modes";
 import { isActive, isDisabled, isEmphasized, isFocused, isHovered, isQuiet, isSelected, size, staticColor } from "@spectrum-css/preview/types";
 import pkgJson from "../package.json";
 import { ActionButtonGroup, ActionButtons } from "./actionbutton.test.js";
+import { ActionButtonsWithIconOptions, TreatmentTemplate } from "./template.js";
 
 /**
  * The action button component represents an action a user can take.
```

Comment:
It looks like there is only one note under the old migration guide that should still be included. I would add this to the hold icon story description: "Because of the way padding is calculated, the hold icon must be placed before the workflow icon in the markup."

The rest of the migration notes would be good to move into the CHANGELOG under the appropriate versions where those notes were added (usually I do a little digging on the git blame in Github to find this).

Reply:
@aramos-adobe If you could move the migration guide to the [action button CHANGELOG](https://github.com/adobe/spectrum-css/blob/main/components/actionbutton/CHANGELOG.md), that'd be great! If/when we stop supporting the static docs site, we don't want to lose all the migration guide stuff you found in the `yml` files, so that's where we've been moving them (from the yml and into the changelog).

And then if we could add Josh's suggestion above for the hold icon docs, that'd be greeaaatttt 🔥